common/supplemental/likelySubtags.xml

```diff
@@ -3730,5 +3730,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; Yi; ? } => { Sichuan Yi; Yi; China }-->
 		<likelySubtag from="und_Zanb" to="cmg_Zanb_MN"/>
 		<!--{ ?; Zanabazar Square; ? } => { Classical Mongolian; Zanabazar Square; Mongolia }-->
+
+		<!-- testing -->
```

Comment:
If so maybe the comment "testing" could be expanded a bit

Reply:
Hm.  @macchiati I see the "not to be patched by hand". Is there a better solution for this file?

Reply:
This file is generated, so the tool needs to be modified for it. However, it would be best to avoid adding "test only" locales to this. Is there a way to avoid adding "mul" here?

Reply:
@macchiati Now that these locales don't exist when tests are run, we might be able to get away with just not mentioning them in likely subtag.  Is there any other machinery that would break  (in checks, etc) if likely subtags aren't present? Worst case I would add exclusions to any such tests.

I'll try this.